README.md

```diff
@@ -170,18 +186,36 @@ With reference to the [GitHub Commit status API], the `POST` parameters (`state`
   The OAuth access token.\
   See also: section [GitHub OAuth token](#github-oauth-token).
 
-## Optional keys
+### Optional keys
 
 - `context_prefix`\
   The prefix for the GitHub Commit status API "context" (see section [Effects on GitHub](#effects-on-github)). If present, the context will be set as `context_prefix/job_name`.\
   Default: empty.\
   See also: the optional `context` in the [put step](#the-put-step).
 
+- `log_level`:\
+  The log level (one of `debug`, `info`, `warn`, `error`, `silent`).\
+  Default: `info`.
+
+- `log_url`. **DEPRECATED, no-op, will be removed**\
+  A Google Hangout Chat webhook. Useful to obtain logging for the `check` step for Concourse < v7.x
+
+
+## GitHub commit status plus chat notifications
+
+### Required keys
+
+- The keys required for [Only GitHub commit status](##github-commit-status-only)
```

Comment:
` The keys required for ... plus the following key/value` 

Maybe I would add also `plus...`

Reply:
It's personal taste, this is a list and you can follow each items . Adding `plus` seems a bit redundant to me.